frontend/src/component/project/Project/PaginatedProjectFeatureToggles/ProjectOnboarding/SdkExample.tsx

```diff
@@ -0,0 +1,72 @@
+import { type SelectChangeEvent, styled, Typography } from '@mui/material';
+import { Link } from 'react-router-dom';
+import Select from '../../../../common/select';
+import { OFFICIAL_SDKS } from '../../../../integrations/IntegrationList/AvailableIntegrations/SDKs';
```

Comment:
I added the SDK list in the onboarding directory. The file is called sharedTypes.ts. We can use the integrations dir but I wanted to control which ones we display in the onboarding just in case the lists don't match between integrations and onboarding.

Reply:
We have value level lists in the SelectSdk.tsx so you can export it from there. Then we don't depend on the other module.